crypto/src/main/scala/org/bitcoins/crypto/CryptoRuntime.scala

```diff
@@ -211,6 +211,17 @@ trait CryptoRuntime {
 
   def tweakMultiply(publicKey: ECPublicKey, tweak: FieldElement): ECPublicKey
 
+  def tweakMultiply(point: SecpPoint, tweak: FieldElement): SecpPoint = {
+    point match {
+      case SecpPointInfinity => SecpPointInfinity
+      case pt: SecpPointFinite =>
+        Try(tweakMultiply(pt.toPublicKey, tweak)) match {
+          case Failure(_)       => SecpPointInfinity
```

Comment:
This really should double check by multiplying by tweak.subtract(Field element.one) and checking they're opposite